### DISCUSSION TITLE
first login failed

### DISCUSSION THREAD

**Discussion by Archi-WeB-tracker** [kimai/kimai#4433] [Questions & Answers]
hi, i have installed and running kimai2 on an synology server dsm 7, with container manager (all green status), i can reach even the login page, but i can't pass  the login, user and password are wrong, in the database_url is noted mysql://kimaiuser/kimaipassword, is that the correct user an pw? it doesen't work for me??? thanks

**Comment by derdeimos** [comment]
Hi,
I have the same issue with kimai2 on Wampserver 3.3.0, MySQL version: 8.0.31. (I followed this steps: https://gist.github.com/FireLemons/f9ff83ba1ccf4b83d835f9b63b57c809)
I can create users on the MySQL console.
And access the page http://localhost/kimai2/public/en_US/login 
With phpmyadmin and MySQL I can see the databases and users.

But with all users that I try, the error "Invalid credentials." appears. and I can not login.

Did you find out how to log in?

**Comment by derdeimos** [comment]
It is in the documentation...

https://www.kimai.org/documentation/installation.html
Create your first user

this command worked for me
`php bin/console kimai:user:create username admin@example.com ROLE_SUPER_ADMIN`
